getgather/mcp/main.py

```diff
@@ -15,6 +15,13 @@
 auto_import("getgather.mcp.brand")
 
 
+# Ensure calendar MCP is registered by importing its module
+try:
+    from getgather.mcp import calendar_utils  # type: ignore
+except Exception as e:
+    logger.warning(f"Failed to register calendar MCP: {e}")
+
+
```

Comment:
Just wondering, why not follow the same pattern as the other MCP registrations?

Reply:
@kpprasa Do you anticipate that auto-registering by module import may throw an exception?